docs/specifications/tests/RequirementsAndNormalizationOfDomainNames.md

```diff
@@ -0,0 +1,476 @@
+[LOW LINE]:                              https://codepoints.net/U+005F
+[MEDIUM MATHEMATICAL SPACE]:             https://codepoints.net/U+205F
+[NO-BREAK SPACE]:                        https://codepoints.net/U+00A0
+[NOTICE]:                                ../SeverityLevelDefinitions.md#notice
+[OGHAM SPACE MARK]:                      https://codepoints.net/U+1680
+[PUNCTUATION SPACE]:                     https://codepoints.net/U+2008
+[RFC 1034#3.1]:                          https://datatracker.ietf.org/doc/html/rfc1034#section-3.1
+[RFC 1034]:                              https://datatracker.ietf.org/doc/html/rfc1034
+[RFC 1035#2.3.3]:                        https://datatracker.ietf.org/doc/html/rfc1035#section-2.3.3
+[RFC 1035#2.3.4]:                        https://datatracker.ietf.org/doc/html/rfc1035#section-2.3.4
+[RFC 1035]:                              https://datatracker.ietf.org/doc/html/rfc1035
+[RFC 1123#2.1]:                          https://datatracker.ietf.org/doc/html/rfc1123#section-2.1
+[RFC 1123]:                              https://datatracker.ietf.org/doc/html/rfc1123
+[RFC 2317#4]:                            https://datatracker.ietf.org/doc/html/rfc2317#section-4
+[RFC 2317]:                              https://datatracker.ietf.org/doc/html/rfc2317
+[RFC 2782]:                              https://datatracker.ietf.org/doc/html/rfc2782
+[RFC 5890#1.1]:                          https://datatracker.ietf.org/doc/html/rfc5890#section-1.1
+[RFC 5890#2.3.2.1]:                      https://datatracker.ietf.org/doc/html/rfc5890#section-2.3.2.1
+[RFC 5890#2.3.2.3]:                      https://datatracker.ietf.org/doc/html/rfc5890#section-2.3.2.3
+[RFC 5890]:                              https://datatracker.ietf.org/doc/html/rfc5890
+[RFC 5895#2]:                            https://datatracker.ietf.org/doc/html/rfc5895#section-2
+[RFC 5895]:                              https://datatracker.ietf.org/doc/html/rfc5895
+[SIX-PER-EM SPACE]:                      https://codepoints.net/U+2006
+[SOLIDUS]:                               https://codepoints.net/U+002F
+[SPACE]:                                 https://codepoints.net/U+0020
+[Severity Level Definitions]:            ../SeverityLevelDefinitions.md
+[Syntax01]:                              ../Syntax-TP/syntax01.md
+[Syntax02]:                              ../Syntax-TP/syntax02.md
+[THIN SPACE]:                            https://codepoints.net/U+2009
+[THREE-PER-EM SPACE]:                    https://codepoints.net/U+2004
+[Unicode SpecialCasing]:                 https://www.unicode.org/Public/UCD/latest/ucd/SpecialCasing.txt
+[Unicode TR 46#Notation]:                http://unicode.org/reports/tr46/#Notation
+[Unicode TR 46]:                         http://unicode.org/reports/tr46
+[Unicode]:                               https://unicode.org/main.html
+[WARNING]:                               ../SeverityLevelDefinitions.md#warning
```

Comment:
I don't see where it's used, and the link should be `./SeverityLevelDefinitions.md#warning`

Reply:
Removed.